packages/docs/content/guidelines/print-guidelines.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: Print guidelines
```

Comment:
I imagine we want this to show up at the bottom of the list in the nav instead of the top, so we should specify an order here like
```suggestion
title: Print guidelines
order: 40
```